examples/resources/squadcast_workflow/resource.tf

```diff
@@ -0,0 +1,61 @@
+resource "squadcast_workflow" "example_workflow" {
+   title = "test workflow"
+   description = "Test workflow description"
+   owner_id = "63bfabae865e9c93cd31756e"
```

Comment:
can you instead use data_source to fetch the id (just be uniform across examples)